.circleci/config.yml

```diff
@@ -178,9 +195,27 @@ jobs:
       - run:
           name: Checking outputs
           command: |
+            # find file
+            find /tmp/data/five-echo/TED.five-echo/* \
+                -exec basename {} \; > /tmp/data/five-echo/TED.five-echo/outputs.out
+            # set filenames
+            f1=/tmp/src/tedana/.circleci/tedana_outputs_verbose.txt
+            f2=/tmp/data/five-echo/TED.five-echo/outputs.out
```

Comment:
Since this file is being generated in the CI, we'll need to remove it when we do the "exact" testing since it's not an expected output of tedana ! That's the source of the current error, I believe.

Reply:
Resolved in f1bc063